'MODULE_NOT_FOUND' error when importing the package in marp

Hello, I use marp with `.mjs` and `.yml` file.
Note that using `@kazumatu981/markdown-it-kroki` works fine, but trying to import `@kazumatu981/markdown-it-fontawesome` gives the following error:

```
node:internal/modules/cjs/loader:1181
  const err = new Error(`Cannot find module '${request}'`);

Error: Cannot find module 'D:\slides\node_modules\markdown-it\lib\rules_block\list'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:1181:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:1169:15)
    at resolveExports (node:internal/modules/cjs/loader:591:14)
    at Module._findPath (node:internal/modules/cjs/loader:668:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1130:27)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (D:\\slides\node_modules\@kazumatu981\markdown-it-fontawesome\dist\lib\FaTagListRuleEngine.js:7:32)
    at Module._compile (node:internal/modules/cjs/loader:1376:14) {
  code: 'MODULE_NOT_FOUND',
  path: 'D:\\slides\\node_modules\\markdown-it\\package.json'
}
```

`.marprc.yml`

```yaml
allowLocalFiles: true
output: "./build"
inputDir: "./slides"
html: true
engine: "./engine.mjs"
template: bespoke
```

`engine.mjs`

```js
import markdownItKroki from '@kazumatu981/markdown-it-kroki';
import markdownItFontawesome from '@kazumatu981/markdown-it-fontawesome';

export default ({ marp }) => {
    
    marp
        .use(markdownItKroki)
        .use(markdownItFontawesome);
        
    return marp;
}
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,0 +1,5 @@
+# CHANGELOG
+
+## 1.0.4
+
+* support [issue](https://github.com/kazumatu981/markdown-it-fontawesome/issues/1)

--- a/lib/FaTagListRuleEngine.ts
+++ b/lib/FaTagListRuleEngine.ts
@@ -1,13 +1,17 @@
-import StateBlock from "markdown-it/lib/rules_block/state_block"
-import list from 'markdown-it/lib/rules_block/list';
+import type StateBlock from "markdown-it/lib/rules_block/state_block"
+import type ParserBlock from "markdown-it/lib/parser_block";
 import { MarkdownItEngineBase } from "./MarkdownItEngineBase";
 import { FaTagListTokenReplacer } from "./FaTagListTokenReplacer";
 import { FontawesomeOption } from "./FontawesomeOption";
 
 export class FaTagListRuleEngine extends MarkdownItEngineBase<FontawesomeOption> {
+    defaultListRule?: ParserBlock.RuleBlock | null;
     jack(state: StateBlock, startLine: number, endLine: number, silent: boolean): boolean {
         const newTokenStart = state.tokens.length;
-        const ret = list(state, startLine, endLine, silent);
+        let ret = false;
+        if(this.defaultListRule){
+            ret = this.defaultListRule(state, startLine, endLine, silent);
+        }
         if (ret) {
             const newTokenEnd = state.tokens.length - 1;
             if (!silent) {
@@ -19,11 +23,21 @@ export class FaTagListRuleEngine extends MarkdownItEngineBase<FontawesomeOption>
     }
 
     use() {
+        this.defaultListRule = this._getDefaultListRule();
         this._md.block.ruler.at(
             'list',
             (state: StateBlock, startLine: number, endLine: number, silent: boolean) => {
                 return this.jack(state, startLine, endLine, silent);
             },
             { alt: ['paragraph', 'reference', 'blockquote'] });
+    }
+
+    _getDefaultListRule(): (ParserBlock.RuleBlock | null) {
+        for(const rule of this._md.block.ruler.getRules('')) {
+            if(rule.name === 'list') {
+                return rule;
+            }
+        }
+        return null;
     }
 }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@kazumatu981/markdown-it-fontawesome",
-  "version": "1.0.3",
+  "version": "1.0.4",
   "description": "markdown-it plugin to be able to use fontawesome",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "@marp-team/marp-cli": "^2.4.0",
     "@tsconfig/node16": "^1.0.3",
     "@types/jest": "^29.5.0",
-    "@types/markdown-it": "^12.2.3",
+    "@types/markdown-it": "^13.0.7",
     "@types/node": "^18.15.10",
     "jest": "^29.5.0",
     "markdown-it-attrs": "^4.1.6",

--- a/tests/unittests/FaTagTokenizer.test.ts
+++ b/tests/unittests/FaTagTokenizer.test.ts
@@ -1,12 +1,12 @@
 import { describe } from '@jest/globals';
-import StateInline from 'markdown-it/lib/rules_inline/state_inline';
+import type StateInline from 'markdown-it/lib/rules_inline/state_inline';
 import MarkdownIt from 'markdown-it';
 import { FaTokenizerBase } from '../../lib/FaTagTokenizer';
 
 function _createStateMock(str: string, pos: number): StateInline {
-    const mockClass = new StateInline(str, new MarkdownIt(), null, []);
-    mockClass.pos = pos;
-    return mockClass;
+    return {
+        src: str, pos:pos
+    } as StateInline;
 }
 
 

